### DISCUSSION TITLE
Dome Water Shutoff Valve Integration question

### DISCUSSION THREAD

**Discussion by stamandr** [jason0x43/hacs-hubitat#221] [Q&A]
Does this integration expose a Dome Water Shutoff Valve as a switch or binary_sensor. I need it to be a switch that can be turned on/off. 
Thanks....

**Comment by jason0x43** [comment]
It depends on what capabilities Hubitat exposes for the device. If you could paste the [device capabilities](https://github.com/jason0x43/hacs-hubitat#checking-device-capabilities) here I can check.

**Reply by mattstein111** [reply]
Mind if I jump in? I'm using a different Zigbee valve, and have the same issue as OP.

Capabilities:

[{"capabilities":["Configuration","Actuator","PowerSource",{"attributes":[{"name":"powerSource","currentValue":"unknown","dataType":"ENUM","values":["battery","dc","mains","unknown"]}]},"Refresh","Battery",{"attributes":[{"name":"battery","currentValue":null,"dataType":"NUMBER","values":null}]},"Valve",{"attributes":[{"name":"valve","currentValue":"open","dataType":"ENUM","values":["open","closed"]}]},"HealthCheck",{"attributes":[{"name":"checkInterval","currentValue":null,"dataType":"NUMBER","values":null}]}]}]

I can open/close the valve from Hubitat, but through the integration all I see in HA is status (and some other irrelevant things) but no ability to open or close.

**Comment by stamandr** [comment]
Right now I am using the Dome Water Shutoff Valve in Smartthings, and using it with the Smartthings integration in Home Assistant.  Smartthings used to expose this as a switch using the custom driver provided by Dome. But now, using the edge drivers it is exposed as a binary_sensor. You can no longer ask Google or Alexa to turn off the water valve due to it being exposed as a binary_sensor instead of a switch.  When I install the Dome Valve directly in Home Assistant it exposes itself as a switch (switch.dome_water_main_shut_off). I don't have a Hubitat device to test with. I am researching this hoping it will expose itself as a switch in Hubitat so I can proceed with my Hubitat purchase. Someone in the Hubitat community must be using the Dome Valve.  I'm specifically interested in using Hubitat with Home Assistant because of Hubitat C-8's Z-Wave and Zigbee radios longer range.

**Comment by jason0x43** [comment]
You might try http://community.hubitat.com to find out if others are using that water valve with Hubitat. If Hubitat exposes a switch for that device (which I’d expect), then it will show up as a switch in the HA integration.

Just out of curiosity, if you already have Home Assistant setup, is there a particular reason you want to get a Hubitat? You can get standalone 800 series zwave adapters.

**Reply by stamandr** [reply]
I have many z-wave and zigbee devices. I like the all in one solution instead of having multiple  hubs. In my current configuration I use Smartthings WiFi Hubs for most of my Water Sensor devices and Home Assistant (Nortek husbzb-1) for others. The reason for this is that Smartthings is not reliable. If Smartthings fails for any reason (and it has) at least HA would pick things on the other devices and let me know.  I'm wanting to get rid of ST and do the same config with Hubitat, that is if Hubitat exposes the Dome Valve as a switch

**Reply by jason0x43** [reply]
HA can directly handle WiFi, Zigbee, and Zwave, and it's a good bit more powerful than ST or Hubitat (although not as simple to get started with).

It looks like Hubitat does [have a built in Dome driver](https://community.hubitat.com/t/dome-water-valve/636/2) that supports control, so you should be good if you want to go that route.

**Comment by stamandr** [comment]
I've been using HA for a number of years and agree with you, it's definitely a beast. I have a complex setup in HA and am very familiar with it. Although HA is great, I don't want all of my eggs in one basket (the more redundancy the better for me), that's why I use a combination of smart home platforms/devices.... Thanks for your suggestions and help
